linkerd/http/box/src/body.rs

```diff
@@ -39,6 +39,13 @@ impl BoxBody {
             inner: Box::pin(Inner(inner)),
         }
     }
+
+    /// Returns an empty [`BoxBody`].
+    ///
+    /// This is an alias for [`BoxBody::default()`].
+    pub fn empty() -> Self {
+        Self::default()
+    }
```

Comment:
i'm curious to see how people feel about this.

i'm not strongly attached to it, but it seems like a friendly alias to expose. it has the small benefit of making it marginally easier to port code from `hyper::body::Body` to the new 1.0 interface.